src/SDKs/SqlManagement/Management.Sql/Generated/SdkInfo_SqlManagementClient.cs

```diff
@@ -94,16 +95,5 @@ internal static partial class SdkInfo
               }.AsEnumerable();
           }
```

Comment:
Please pull latest remote changes. run `msbuild build.proj` and regenerate the code

Reply:
Done